Vagrantfile

```diff
@@ -0,0 +1,119 @@
+# -*- mode: ruby -*-
+
+VAGRANTFILE_API_VERSION = "2"
+
+# Required Vagrant plugins.
+['vagrant-reload', 'vagrant-disksize', 'vagrant-vbguest'].each do |plugin|
+  unless Vagrant.has_plugin?(plugin)
+    raise "Vagrant plugin #{plugin} is not installed!"
+  end
+end
+
+# Default Vagrant vars.
+$devbind_img = "capsule/dpdk-devbind:18.11.6"
+$dpdkmod_img = "capsule/dpdk-mod:18.11.6-`uname -r`"
+$sandbox_img = "capsule/sandbox:18.11.6-1.42"
+
+$dpdk_driver = "uio_pci_generic"
+$dpdk_devices = "0000:00:08.0"
```

Comment:
we need a second interface. the `nat64` example uses 2.

```
[[ports]]
    name = "eth1"
    device = "0000:00:08.0"
    cores = [1]

[[ports]]
    name = "eth2"
    device = "0000:00:09.0"
    cores = [1]
```

Reply:
Done did.